.rubocop.yml

```diff
@@ -20,6 +20,9 @@ Naming/PredicateName:
     - def_node_matcher
     - def_node_search
 
+Naming/RescuedExceptionsVariableName:
+  PreferredName: exception
```

Comment:
This setting to .rubocop.yml is unnecessary. There is no reason to be out of default.yml.

Reply:
Yep, let's have `e` as the default for RuboCop as well.